src/lib/support/Span.h

```diff
@@ -179,8 +221,11 @@ class FixedSpan
     template <class U,
               typename = std::enable_if_t<std::is_pointer<U>::value && sizeof(std::remove_pointer_t<U>) == sizeof(T) &&
                                           std::is_convertible<U, T *>::value>>
-    constexpr explicit FixedSpan(U databuf) : mDataBuf(databuf)
-    {}
+    explicit FixedSpan(U databuf) : mDataBuf(databuf)
```

Comment:
The existing comment elsewhere in the file is
```
    // VerifyOrDie cannot be used inside a constexpr function, because it uses
    // "static" on some platforms (e.g. when CHIP_PW_TOKENIZER_LOGGING is true)
    // and that's not allowed in constexpr functions.
```

I'll add a short one-liner to the usages to reference this.